cmd/oras/push.go

```diff
@@ -262,7 +261,9 @@ func pushArtifact(dst *remote.Repository, pack packFunc, packOpts *oras.PackOpti
 	if err := display.PrintStatus(root, "Fallback ", verbose); err != nil {
 		return ocispec.Descriptor{}, err
 	}
-	dst.SetReferrersCapability(false)
+	if repo, ok := dst.(*remote.Repository); ok {
+		repo.SetReferrersCapability(false)
```

Comment:
I may have missed something... Can you help me to understand why do we set the referrers capability here?

Reply:
Since it's fallbacking, we assumes **Referrers API is not supported** based on **artifact media type is not supported**. Let me add doc in code.